src/index.html

```diff
@@ -1,22 +1,99 @@
 <!doctype html>
-<html lang="en">
+<html lang="pt-BR">
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Moyo Header</title>
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="../src/images/logo.png"
+          alt="logo"
+          aria-label="Go to homepage"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list list">
+          <li class="list__item">
+            <a
+              class="list__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              class="list__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              class="list__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="list__item"
+            data-qa="hover"
+          >
+            <a
+              class="list__link la__link"
```

Comment:
There is no need for this second class `la__link`, this link works the same as the others, but it has another property called data-qa="hover"